### PR TITLE
[IMP] account,l10n_*: default deferral accounts

### PR DESCRIPTION
**Pull request by dylankiss**
*ae,au,hk,ke,mx

We want to set up sensible default accounts for the deferred expenses and deferred revenues in different localizations.

[task-3419066](https://www.odoo.com/web#id=3419066&cids=1&menu_id=4720&action=333&active_id=967&model=project.task&view_type=form)

Related to https://github.com/odoo/enterprise/pull/46383

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/chart_template.py [hunk 1]
```diff
@@ -437,7 +438,7 @@ def deref(values, model):
                         values[fname] = self.ref(value).id if value not in ('', 'False', 'None') else False
                     except ValueError as e:
                         _logger.warning("Failed when trying to recover %s for field=%s", value, field)
-                        raise e
+                        failed_fields.append(fname)
```
Comment (william-andre) [inline addons/account/models/chart_template.py:441]: Can't we just `del values[fname]` here?

Comment (dylankiss) [reply addons/account/models/chart_template.py:441]: Tried that first as well, but Python is being too smart, with `fields` actually looping over `values` and it doesn't allow changing the size during iteration:

```
...
  File "/Users/dylankiss/src/odoo-multiverse/master/odoo/addons/account/models/chart_template.py", line 521, in _load_data
    'values': deref(record, self.env[model]),
  File "/Users/dylankiss/src/odoo-multiverse/master/odoo/addons/account/models/chart_template.py", line 430, in deref
    for field, fname, value in fields:
  File "/Users/dylankiss/src/odoo-multiverse/master/odoo/addons/account/models/chart_template.py", line 428, in <genexpr>
    fields = ((model._fields[k], k, v) for k, v in values.items() if k in model._fields)
RuntimeError: dictionary changed size during iteration
...
```

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by william-andre** [comment]
@robodoo rebase-ff

**Pull request closed without merging**